python/examples/ola_artnet_params.py

```diff
@@ -26,9 +26,9 @@
 def ArtNetConfigureReply(state, response):
   reply = ArtNetConfigMessages_pb2.Reply()
   reply.ParseFromString(response)
-  print 'Short Name: %s' % reply.options.short_name
-  print 'Long Name: %s' % reply.options.long_name
-  print 'Subnet: %d' % reply.options.subnet
+  print(('Short Name: %s' % reply.options.short_name))
```

Comment:
As mentioned in the corresponding commit message, this was done automatically using the 2to3 tool. Going through the whole patch to remove the superfluous parentheses would be time consuming, but could be done. (any idea on how to do it programmatically?)

I think the tool is being overly safe, adding parentheses to wrap the % operation and some for the print.